static/js/components/dashboard/FinalExamCard_test.js

```diff
@@ -138,4 +152,34 @@ pay for the course and pass the online work.`;
       'ERROR ERROR'
     );
   });
+
+  it('renders confirm pearson TOS dialog', () => {
+    props.ui.dialogVisibility = { 'pearsonTosDialogVisible': true };
+    renderCard(props);
```

Comment:
sorry I don't mean to be confusing!

to fix these tests you just need to put this before the render call:

```js
props.program.pearson_exam_status = PEARSON_PROFILE_SCHEDULABLE;
```